languages/csharp/exercises/concept/properties/PropertiesTests.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using Xunit;
+
+public class WeighingMachineTests
+{
+    [Fact]
+    public void Set_weight_and_get_weight()
+    {
+        var wm = new WeighingMachine();
+        wm.InputWeight = 60m;
+        Assert.Equal(60m, wm.InputWeight, 3 );
```

Comment:
Maybe use a named parameter here as it might not be obvious to everyone what the third parameter is (great idea to use that BTW):

```suggestion
        Assert.Equal(60m, wm.InputWeight, precision: 3);
```

Reply:
very true